include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_fpga.h

```diff
@@ -76,13 +76,23 @@ struct __parallel_for_fpga_submitter<__internal::__optional_kernel_name<_Name...
     }
 };
 
+template <typename _ExecutionPolicy>
+auto
+__device_policy(_ExecutionPolicy&& __exec)
+    -> decltype(oneapi::dpl::execution::make_device_policy<
+                oneapi::dpl::execution::policy_kernel_name<_ExecutionPolicy>>(__exec.queue()))
```

Comment:
Is it required to list the type here?  With such a small function, it seems redundant unless it is required for some reason I am missing.

Reply:
Fixed